modules/@apostrophecms/notification/ui/apos/apps/AposNotification.js

```diff
@@ -10,4 +11,8 @@ export default function() {
   }
   const app = createApp(component);
   app.mount(el);
+
+  const notifStore = useNotificationStore();
+  apos.notify = notifStore.notify;
+  notifStore.poll();
```

Comment:
Start notification polling from here. Could be done in store mounted, both sounds ok to me.